configs/_base_/default_runtime.py

```diff
@@ -19,3 +19,8 @@
 opencv_num_threads = 0
 # set multi-process start method as `fork` to speed up the training
 mp_start_method = 'fork'
+
+# In order to support automatically scaling LR. Default batch size
```

Comment:
The default batch size of configs in MMDet. This key allows to scale LR automatically.

Reply:
Done

Reply:
Done